src/OpenLoco/Town.cpp

```diff
@@ -63,9 +149,22 @@ namespace OpenLoco
      */
     void Town::recalculateSize()
     {
-        registers regs;
-        regs.esi = X86Pointer(this);
-        call(0x004975E0, regs);
+        auto newSize = enumValue(size);
+        if (size < TownSize::metropolis
+            && var_34 >= _populations[enumValue(size) + 1])
```

Comment:
the `&&` doesn't need to be on a new line

Reply:
layout is handled by clang format. I tend to not fight it too much.

Reply:
hmm interesting, I manually formatted it to be a single line, saved in VS (and so clang format ran) and nothing changed, it was happy with it. it's strange it put this on a new line when there are if statements close by that are longer in length. anyways yeh, if clang format did that for you then we just leave it